SayTheirNames/Source/Controller/More/MoreView.swift

```diff
@@ -84,119 +143,76 @@ final class MoreView: UIView {
+        let label = UILabel()
+        label.font = UIFont.STN.ctaTitle
+        label.textColor = STNAsset.Color.primaryLabel.color
+        label.numberOfLines = Theme.Components.LineLimit.double
+        label.translatesAutoresizingMaskIntoConstraints = false
         
         switch section {
         case .history:
-            titleLabel.text = Strings.MoreHistory.title
+            label.text = Strings.MoreHistory.aboutTitle
+        case .contribution:
+            label.text = Strings.GetInvolved.Slack.title
         case .developer:
-            titleLabel.text = Strings.GetInvolvedDev.title
-        case .designer:
-            titleLabel.text = Strings.GetInvolvedDesign.title
+            label.text = Strings.GetInvolved.Developer.title
+        case .twitter:
+            label.text = Strings.GetInvolved.Twitter.title
         }
-        return titleLabel
+        
+        return label
     }
     
-    private func descriptionLabel(_ section: section) -> UILabel {
-        let bodyLabel = UILabel()
-        
-        bodyLabel.translatesAutoresizingMaskIntoConstraints = false
-        bodyLabel.textColor = STNAsset.Color.primaryLabel.color
-        bodyLabel.font = UIFont.STN.ctaBody
-        bodyLabel.numberOfLines = Theme.Components.LineLimit.quintuple
-        
-        switch section {
-        case .history:
-            bodyLabel.text = """
-            At vero eos et accusamus et iusto odio dignissimos ducimus qui blanditiis praesentium
-            voluptatum deleniti atque corrupti quos dolores et quas molestias excepturiasdlnalkd.
-            """
-        case .developer:
-            bodyLabel.text = "At vero eos et accusamus et iusto odio dignissimos ducimus qui blanditiis"
-        case .designer:
-            bodyLabel.text = "At vero eos et accusamus et iusto odio dignissimos ducimus qui blanditiis"
-        }
-        
-        return bodyLabel
+    func getDescriptionLabel(for section: MoreSection, with description: String = "") -> UILabel {
+        let label = UILabel()
+        label.text = description
+        label.font = UIFont.STN.ctaBody
+        label.textColor = STNAsset.Color.primaryLabel.color
+        label.numberOfLines = Theme.Components.LineLimit.none
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
     }
     
-    private func actionButton(_ section: section) -> UIButton {
-        let actionButton = UIButton()
-        actionButton.translatesAutoresizingMaskIntoConstraints = false
-
-        actionButton.titleLabel?.font = UIFont.STN.sectionHeader
-        actionButton.layer.borderWidth = 2
-        actionButton.backgroundColor = STNAsset.Color.actionButton.color
-        actionButton.setTitleColor(STNAsset.Color.actionButtonTint.color, for: .normal)
-        actionButton.heightAnchor.constraint(equalToConstant: 50).isActive = true
+    func getActionButton(for section: MoreSection) -> UIButton {
```

Comment:
super minor - can be private, and naming-wise `makeActionButton` makes more sense because it creates a new instance every time.

Reply:
I think all these functions are within private extensions; Proceeding to update the function names!